proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/biometry/IAuthenticateWithBiometryListener.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getlime.security.powerauth.biometry;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.UiThread;
+import io.getlime.security.powerauth.exception.PowerAuthErrorException;
+import io.getlime.security.powerauth.sdk.PowerAuthAuthentication;
+
+/**
+ * Interface used as a callback for biometric authentication for general application use.
+ */
+public interface IAuthenticateWithBiometryListener {
```

Comment:
I think we should use `biometrics` instead of biometry -> that's the feedback we received from our text proofreader, that we're using this in the wrong way.

Reply:
I see you're using it right in the new ENUMs, so we should generally use it properly on all new APIs.

Reply:
OK, then it makes sense to rename the new public method to `authenticateWithBiometrics()` too